gillespy2/core/functiondefinition.py

```diff
@@ -35,7 +38,7 @@ def sanitized_function(self, species_mappings, parameter_mappings):
                        reverse=True)
         replacements = [parameter_mappings[name] if name in parameter_mappings else species_mappings[name]
                         for name in names]
-        sanitized_function = self.function
+        sanitized_function = self.function_string
```

Comment:
`self.function` should be the sanitized function.  `self.function_string` is only used by StochSS

Reply:
Fixed.